tests/test_params_repr.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from fastapi import FastAPI
+from fastapi.params import Body, Depends, Param, ParamTypes
+
+
+@pytest.fixture(scope="class")
+def depends():
+    return Depends()
+
+
+@pytest.fixture(scope="class")
+def depends_args():
+    return Depends(False, use_cache=False)
+
+
+@pytest.fixture(scope="class")
+def body():
+    return Body(1)
+
+
+@pytest.fixture(scope="class")
+def param():
+    return Param(1)
+
+
+@pytest.fixture(scope="class", params=["query", "header", "path", "cookie"])
+def param_type(request):
+    return ParamTypes(request.param), request.param
```

Comment:
After removing the `__repr__` for `ParamTypes` we can remove this fixture and the test below.

![44ws71](https://user-images.githubusercontent.com/1326112/84522229-8aceaa00-acd6-11ea-8f5a-68543cc6c753.jpg)

:wink: :joy: